testsuite/features/support/env.rb

```diff
@@ -150,7 +150,12 @@ def capybara_register_driver
   log "This scenario took: #{current_epoch - @scenario_start_time} seconds"
   if scenario.failed?
     begin
-      if web_session_is_active?
+      if scenario.exception.is_a?(Selenium::WebDriver::Error::WebDriverError)
```

Comment:
You could move this part in restart_webdriver method. But it's also fine like that.

Reply:
YAGNI :smile_cat: , Don't create a method that would be called only once.